src/results.rs

```diff
@@ -11,6 +12,23 @@ use std::str::FromStr;
 use toolchain::Toolchain;
 use util;
 
+pub trait ExperimentResultDB {
```

Comment:
i'm not a fan of this name, but i can't think of a better one right now, so might as well keep it, i dunno